quizapp/src/main/java/billennium/quizapp/webConfig/WebConfig.java

```diff
@@ -0,0 +1,25 @@
+package billennium.quizapp.webConfig;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
```

Comment:
@EnableWebMvc

Reply:
Jest bardzo ciekawy temat związany z tą adnotacją. Odnosi się głównie do tego czy ta adnotacja powinna tu być czy nie (Wg mnie nie). https://dzone.com/articles/spring-boot-enablewebmvc-and-common-use-cases